Assets/MRTK/SDK/Experimental/TouchSlider/TouchSlider.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using TMPro;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// This control enables touching a slider with one finger to set its value
+    /// </summary>
+    public class TouchSlider : MonoBehaviour, IMixedRealityTouchHandler
+    {
+        [Experimental]
+        [SerializeField]
+        [Tooltip("The pinch slider that will be receiving touch input")]
+        private PinchSlider slider = null;
+
+        [SerializeField]
+        [Tooltip("Optional TextMeshPro that displays the slider value to the end user")]
+        private TextMeshPro sliderValue = null;
```

Comment:
I understand this is experimental so this may or may not be a nit. We haven't been including labels in the slider controls themselves. For example, if you look at the **HandInteractionExamples** scene you'll see 3 `PinchSlider` instances. While there are labels for each slider, they are not part of the slider prefab. They are actually stand-alone labels with `ShowSliderValue` scripts attached to monitor changes to the `PinchSlider`. This approach allows more flexible UI design. For example, the **Platonic** object in the scene has a `SliderChangeColor` script attached to it, allowing 3 sliders to change its color.